internal/imgutil/imgutil.go

```diff
@@ -34,7 +35,9 @@ func GetRemoteImage(imgRef string) (v1.Image, error) {
 // ExtractEnvbuilderFromImage reads the image located at imgRef and extracts
 // MagicBinaryLocation to destPath.
 func ExtractEnvbuilderFromImage(ctx context.Context, imgRef, destPath string) error {
-	needle := filepath.Clean(constants.MagicBinaryLocation)[1:] // skip leading '/'
+	var o eboptions.Options
+	o.SetDefaults()
```

Comment:
(envbuilder review: Usage here might be nicer with `o := eboptions.Defaults()`.)